### DISCUSSION TITLE
Convert Single Image to Gif with 5 frames

### DISCUSSION THREAD

**Discussion by skyemk** [ImageMagick/ImageMagick#4122] [Help] [answered]
I know how to convert formats to gif.

But how do I convert 1 picture to gif that has 5 frames of the same picture?

**Comment by fmw42** [comment, marked as answer]
`convert image.suffix -duplicate 4 new_image.gif`

**Reply by fmw42** [reply]
What is your ImageMagick version and platform? You may be on an older version of IM that does not have -duplicate. Also if using IM 7, then change convert to magick.

**Reply by skyemk** [reply]
oh god, sorry about this late century reply. I gave up on what I tried to do months back. If it helps ill mark urs as answer to close this.
